public/pages/ThreatIntel/containers/AddThreatIntelSource/AddThreatIntelSource.tsx

```diff
@@ -203,8 +367,18 @@ export const AddThreatIntelSource: React.FC<AddThreatIntelSourceProps> = ({
           <h4>Details</h4>
         </EuiText>
         <EuiSpacer />
-        <EuiFormRow label="Name">
-          <EuiFieldText placeholder="Title" onChange={onNameChange} value={source.name} />
+        <EuiFormRow
+          label="Name"
+          helpText="Rule name must contain 1-128 characters. Valid characters are a-z, A-Z, 0-9, hyphens, spaces, and underscores."
```

Comment:
Little tough to visualize where this text will appear in the UI, but should this say "Rule name"?

Reply:
Nope, great catch!